src/components/Heading/Heading.tsx

```diff
@@ -1,14 +1,15 @@
 import * as React from 'react';
 import { themr, ThemedComponentClass } from '@friendsofreactjs/react-css-themr';
-import { classNames } from '@shopify/react-utilities/styles';
 import { HeadingTagName } from '../../types';
-
+import { classNames, variationName } from '@shopify/react-utilities/styles';
 import { HEADING } from '../ThemeIdentifiers';
 import * as baseTheme from './Heading.scss';
-
+export type Color = 'text' | 'darker' | 'darkest' | 'mid' | 'reverse' | 'danger';
```

Comment:
@Farhan-emgage let's talk about this. I'm not convinced that Headings should have text color options.